generator/osm2meta.cpp

```diff
@@ -102,22 +102,7 @@ std::string MetadataTagProcessorImpl::ValidateAndFormat_stars(std::string const
 
 std::string MetadataTagProcessorImpl::ValidateAndFormat_operator(std::string const & v) const
 {
-  auto const & t = m_params.m_types;
-  if (ftypes::IsATMChecker::Instance()(t) ||
-      ftypes::IsPaymentTerminalChecker::Instance()(t) ||
-      ftypes::IsMoneyExchangeChecker::Instance()(t) ||
-      ftypes::IsFuelStationChecker::Instance()(t) ||
-      ftypes::IsRecyclingCentreChecker::Instance()(t) ||
-      ftypes::IsPostOfficeChecker::Instance()(t) ||
-      ftypes::IsCarSharingChecker::Instance()(t) ||
-      ftypes::IsCarRentalChecker::Instance()(t) ||
-      ftypes::IsBicycleRentalChecker::Instance()(t) ||
-      ftypes::IsParkingChecker::Instance()(t))
-  {
-    return v;
-  }
-
-  return {};
+  return v;
```

Comment:
I have doubts here. There will be too many additional _useless_ (from my perspective) strings. Who cares about "operator" for power=* ? The same is for "network". Who cares about the highway's network? I think we should keep the existing amenity (POI) checker. Ok, probably add PT.

https://taginfo.openstreetmap.org/keys/operator#combinations
https://taginfo.openstreetmap.org/keys/network#combinations

Reply:
I reverted back to the previous but added some more tags to operator validator